### ISSUE TITLE
Error Messages for Volunteers via SMS

### ISSUE THREAD

**Issue by guddukaur** [IFRCGo/cbs#719]
Hi there team,

Pilot Country has come back to us with an issue to be resolved. SMS messages which do not meet reporting format requirements are being sent to volunteers with no feedback. Can we please request than an Error Message be sent to Volunteers from the Platform informing them that the format is incorrect and what the correct format is.  

This could look like:

Thank you for reporting. The message format is incorrect. Please send in X#X#X format 

Thanks,

Guddu

**Comment by alexanderhoset** [comment]
@woksin is the feedback message hard coded in the gateway for now?

**Comment by woksin** [comment]
@alexanderhoset Yes, as far as I know the SMSEagle has just been set up to respond with a generic response. There is nothing going on in the CBS system at the moment that generates a feedback for a datacollector by sms. This is certainlly something that we have in our todo list, but as it is now I don't know if we can communicate with the gateway so that we can send feedback. And I don't think that we have a working API for this yet. @eprom or @einari should know more about this.

**Comment by alexanderhoset** [comment]
if we solve  #275 and #507 we have this feature and can close this
